<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>20</y>
      <width>531</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>231</width>
      <height>571</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Управление расписанием</string>
    </property>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить из файла(.txt)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск по номеру рейса</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sort_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>qsort</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inform_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>info</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="filtr1_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Фильтр 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="filtr2_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>231</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Фильтр 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="acceptFiltr_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Применить фильтр</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>211</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="flightNumber">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>№ рейса</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="planeType">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип самолета</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="destination">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пункт назн.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Вр. вылета</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="time">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
